code/parachain/runtime/picasso/src/tracks.rs

```diff
@@ -22,36 +22,40 @@ const fn percent(x: i32) -> sp_runtime::FixedI64 {
 	sp_runtime::FixedI64::from_rational(x as u128, 100)
 }
 
+const fn permill(x: i32) -> sp_runtime::FixedI64 {
+	sp_runtime::FixedI64::from_rational(x as u128, 1000)
```

Comment:
Why not to use the standard `Permill` type?

Reply:
i dont think it suport FixedI64